tests/test_back_filters.js

```diff
@@ -12,7 +12,8 @@ const originalMode = nockBack.currentMode
 nock.enableNetConnect()
 nockBack.fixtures = `${__dirname}/fixtures`
 nockBack.setMode('record')
-const fixture = `${nockBack.fixtures}/recording_test.json`
+const fixtureFilename = `recording_filters_test.json`;
+const fixture = `${nockBack.fixtures}/${fixtureFilename}`
```

Comment:
This is the actual change that fixed the flapping. Changing the filename to something other than the file also being used by `test/test_back_2.js`.

The rest of the PR  (expect the `getTime` change below) is just refactoring and typos.